### ISSUE TITLE
Example Swift code can not contain string interpolation

### ISSUE THREAD

**Issue by blochberger** [realm/jazzy#630]
Documenting a function providing example code renders incorrectly if the example code contains string interpolation:

`````` swift
/**
    #### Example
    ```swift
    let result = doSomething()
    print("Result: \(result)")
    ```
*/
func doSomething() -> String {
    …
}
``````

Which will render as:

> **Example**
> 
> ``` swift
> let result = doSomething()
> print(“Result: ”se“>\(”n“>result”se“>)”s“>”)
> ```

Where the quotation is done by HTML quotes:

``` html
<span class="nf">print</span><span class="p">(</span><span class="s"><q>Result: </q></span><span class="&lt;/q">se<q>&gt;\(</q></span><span class="&lt;/q">n<q>&gt;result</q></span><span class="&lt;/q">se<q>&gt;)</q></span><span class="&lt;/q">s<q>&gt;</q></span><span class="p">)</span>
</code>
```

This issue does not arise when using i.e. `README.md` as abstract. There the quotation marks are rendered as characters as appearing in the source code `"`.

This might be related to #628.

**Comment by istx25** [comment]
I believe this is related to #628. Closing as a duplicate. Will reference this issue in the future where necessary.

**Comment by jpsim** [comment]
Hmm, not obvious to me that this is related to #628. Unless you're certain, @istx25 can you reopen this one?

**Comment by istx25** [comment]
Reopened just in case. I'll do some more digging in the near future.

**Comment by 1ec5** [comment]
#714 is one contributing factor to the unexpected behavior reported in this issue.

**Comment by johnfairh** [comment]
Culprit was the preceding header - fixed by #766 in 0.8.0.
